tests/test_store.py

```diff
@@ -0,0 +1,335 @@
+
+        log.debug(f"found in {ns=} --> {len(results)=}")
+        assert len(results) >= len(nums)
+
+        for i in nums:
+            assert (
+                tuple(
+                    f"https://example.org/{part}#{i}"
+                    for part in ["subject", "predicate", "object"]
+                )
+                in results
+            ), f"expected triple for index { i } not found in result"
+
+    for plan in plans:
+        assert rdf_store.verify_max_age(
+            ns, 1
+        ), "graphs should be inserted and checked in less then a minute"
+
+    sleep(60)  # hey, seriuosly? wait a minute!
+    for plan in plans:
+        assert not rdf_store.verify_max_age(
+            ns, 1
+        ), "after a minute of nothing, those should be older then a minute"
+
+    # now drop the second graph, and check for the (should be none!) effect on the first
+    ns1, nums1 = plans[0]["ns"], plans[0]["nums"]
+    ns2 = plans[1]["ns"]
+    rdf_store.drop_graph(ns2)
+    assert rdf_store.verify_max_age(
+        ns2, 1
+    ), "dropped graph should be marked as changed again"
+
+    # there should be nothing left in ns2
+    try:
+        result = rdf_store.select(sparql, ns2)
+    except (
+        Exception
+    ):  # accept that this could also throw an exception since the graph was dropped!
+        result = []
+    assert len(result) == 0, "there should be no results in a dropped ns"
+
+    # there should be nothing changed in ns1
+    result: List[Tuple[str]] = [
+        tuple(str(u) for u in r) for r in rdf_store.select(sparql, ns1)
+    ]
+    assert len(result) == len(
+        nums1
+    ), "there should still be same results in the kept ns"
+    for i in nums1:
+        assert (
+            tuple(
+                f"https://example.org/{part}#{i}"
+                for part in ["subject", "predicate", "object"]
+            )
+            in results
+        ), f"expected triple for index { i } not found in result of {ns1=}"
+
+    # all stuff should still be there in the overall search-graph
+    result: List[Tuple[str]] = [
+        tuple(str(u) for u in r) for r in rdf_store.select(sparql)
+    ]
+    assert len(result) >= len(
+        nums1
+    ), "there should be at least same results in the overall store"
+    for i in nums1:
+        assert (
+            tuple(
+                f"https://example.org/{part}#{i}"
+                for part in ["subject", "predicate", "object"]
+            )
+            in results
+        ), f"expected triple for index { i } not found in result of overall search"
+
+
+@pytest.mark.usefixtures("prepopulated_rdf_store")
+def test_select_property_trajectory(prepopulated_rdf_store: RDFStore):
+    # SPARQL to select trajectory of a property
+    sparql = """
+    SELECT ?s ?o WHERE {?s <http://purl.org/dc/terms/abstract>/<http://purl.org/dc/terms/else> ?o .}
+    """
+
+    prepopulated_rdf_store.select(sparql)
+    # we should just get here without an error
+    assert True
```

Comment:
strange, no?
maybe at least assert the query-result is not none?
or (even better) make sure the prepopulated store has triples on board that guarantee some useful response-count to be checked for?